exercises/practice/knapsack/.meta/example.el

```diff
@@ -0,0 +1,23 @@
+;;; knapsack.el --- Knapsack (exercism)  -*- lexical-binding: t; -*-
+
+;;; Commentary:
+
+;;; Code:
+
+(require 'cl-lib)
+
+(defun knapsack (capacity weights values)
```

Comment:
Did the exercise generator use `knapsack` for the function name? Looking at problem specifications, It should be `maximum-value`.

Also, the current argument list clashes with the instructions. The docs indicate the items should be a sequence of items with both weight and value stored together. I'd recommend taking a look at the Common Lisp implementation of this exercise.

I'd prefer the function to also be named maximumValue as indicated in problem-specificatins

Reply:
I had changed the name and argument list to follow the Scheme track.

Now updated to follow problem specifications / Common Lisp.